TeamTracker/app/src/main/res/values/strings.xml

```diff
@@ -5,4 +5,10 @@
     <string name="login_password_hint">Password</string>
     <string name="homeScreenTitle">Your Projects</string>
     <string name="project_name">First Project Name Which I Don\'t Know</string>
```

Comment:
Need to change the content I guess

Reply:
Okay. In next commit I will resolve these. For now, this is minor issue. Thanks.